buddyboss/buddyboss-platform
# Group type is disappearing when an organizer changes any group settings with a profile type

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1342

### How to test the changes in this Pull Request:

1. Create a Group Type ex: School from **BuddyBoss > Groups > Group Types**: https://prnt.sc/txtpmx
2. Assign a group type to a group. Make sure the Group Organizer is not an admin: https://prnt.sc/txtr0y
3. Create a Profile Type ex: Student from **BuddyBoss > Profiles > Profiles Types**
4. Set **None** to the **Group Type Creation**: https://prnt.sc/txttyx
5. Assign a profile type to that Organizer: https://prnt.sc/txtus5
6. Go to the group setting as the Organizer.
7. Try to edit any setting: https://prnt.sc/txtwgu
8. See the group type is not there: https://prnt.sc/txtxkl 

### Proof Screenshots or Video
http://somup.com/cYjl1y649o


### Other information:
